CMakeLists.txt

```diff
@@ -211,3 +211,11 @@ install (TARGETS OpenCL
     RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
     ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
     LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR})
+
+if (UNIX)
```

Comment:
Confirming: Do we only want to do this for UNIX?  Or should we do this for other operating systems also?

Reply:
I was unsure if these could successfully be installed for a regular Windows build so erred on the side of caution here. 
I found this to confirm my bias:
https://github.com/pocl/pocl/blob/63b28753749d16209db54614e6a957bd5a203087/CMakeLists.txt#L1720-L1726
but we may reevaluate. This `UNIX` variable should cover Linux, MacOS, cygwin. I am unsure about MinGW and Android.

Reply:
PkgConfig files can be used on Windows (though it's somewhat uncommon). https://sourceforge.net/projects/pkgconfiglite/ can be used on Windows to search and use them.

Reply:
@jenatali thanks for confirming. Is there a convention we need to follow as far as the install location is concerned?

Reply:
No, generally whoever is building a PkgConfig enabled project will specify the install prefix on the command line. From there, normal UNIX subdirectories are expected.

Reply:
For now, according to @mmuetzel suggestion, I deactivated pkg-config support when MSVC is detected.